db-tests/src/db_tests.rs

```diff
@@ -651,12 +651,12 @@ async fn get_collection_timestamps() -> Result<()> {
 async fn get_collection_usage() -> Result<()> {
     let db = db().await?;
 
-    let uid = 1;
+    let uid = 5;
     let mut expected = HashMap::new();
     let mut rng = thread_rng();
 
     for &coll in ["bookmarks", "history", "prefs"].iter() {
-        for i in 0..100 {
+        for i in 0..5 {
```

Comment:
? IIUC, the reduction is because the tests take too long. Maybe declare a single `const` that contains the `MAX_TEST_ITERATION` and use that so you're not hunting these down later?